test/integration_test.py

```diff
@@ -814,22 +814,22 @@ def _source_ids_from_hits(hits: JSONs) -> Set[str]:
                        args={
                            'catalog': catalog,
                            'filters': json.dumps(source_filter)
-                       }).url
-            response = self._get_url_json(url)
+                       })
+            response = self._get_url_json(str(url))
```

Comment:
The change to `url` seems unnecessary for this PR and doesn't match how `summary_url` is composed and used soon after.